proof/verifier.go

```diff
@@ -325,6 +325,32 @@ func (p *Proof) verifyGenesisReveal() error {
 
 	return nil
 }
+
+// verifyGroupKeyReveal verifies that the group key reveal can be used to derive
+// the same key as the group key specified for the asset.
+func (p *Proof) verifyGroupKeyReveal() error {
+	groupKey := p.Asset.GroupKey
+	if groupKey == nil {
+		return ErrGroupKeyRequired
+	}
+
+	reveal := p.GroupKeyReveal
+	if reveal == nil {
+		return ErrGroupKeyRevealRequired
+	}
+
+	revealedKey, err := reveal.GroupPubKey(p.Asset.ID())
```

Comment:
I think this method is missing from the diff?

Reply:
https://github.com/lightninglabs/taproot-assets/pull/448/commits/7a9a80795211934b0658b68a1351c1a1dc5e1e4e#diff-98586afaee9ef4f1d46ba2b4af7ce1fd31d28bc83e6df2117e6483532221f8a7R457